src/test/parsers/minecraft/coord/coord_base.test.ts

```diff
@@ -1,171 +1,135 @@
-                                        }
-                                    }
-                                ],
-                                out.errors
-                            ),
-                        m: "return a float error",
-                        v: ["1.3 1"]
-                    }
-                ]
-            },
-            {
-                s: {
-                    count: 3,
-                    float: true,
-                    local: true
-                },
-                v: [
-                    {
-                        f: out => assert.ok(isSuccessful(out)),
-                        m: "work",
-                        v: [
-                            "1 2 3",
-                            "~1 ~2 3",
-                            "^1 ^ ^2",
-                            "~ ~ ~",
-                            "1.2 3 ~1",
-                            "^.1 ^ ^3"
-                        ]
-                    }
-                ]
-            },
-            {
-                s: {
-                    count: 2,
-                    float: true,
-                    local: false
-                },
-                v: [
-                    {
-                        f: out => assert.ok(isSuccessful(out)),
-                        m: "work",
-                        v: ["~1 ~", "2 3", "5 ~20", "~ ~"]
-                    },
-                    {
-                        f: out =>
-                            assertErrors(
-                                [
-                                    {
-                                        code: "argument.pos.nolocal",
-                                        range: {
-                                            end: 1,
-                                            start: 0
-                                        }
-                                    }
-                                ],
-                                out.errors
-                            ),
-                        m: "return a no local error",
-                        v: ["^ ^3"]
-                    }
-                ]
-            }
-        ] as Array<{
-            s: CoordRules;
-            v: Array<{
-                m: string;
-                v: string[];
-                f(v: ReturnedInfo<undefined>): undefined;
-            }>;
-        }>).forEach(v =>
-            describe(`settings: ${toString(v.s)}`, () =>
-                v.v.forEach(t =>
-                    t.v.forEach(c =>
-                        it(`should ${t.m} for coord '${c}'`, () => {
-                            const parser = new CoordBaseParser(v.s);
-                            const out = parser.parse(new StringReader(c));
-                            t.f(out);
-                        })
-                    )
-                ))
-        );
+        describe("settings: {count: 2, float: false, local: true }", () => {
+            const parser = new CoordParser({
+                count: 2,
```

Comment:
These tests are only showing what you want them to show. They need to show all aspects of the parser working properly, which is provided by the `testParser` api